suite-native/module-settings/src/slice.ts

```diff
@@ -7,6 +7,7 @@ export interface AppSettingsState {
     isOnboardingFinished: boolean;
     fiatCurrency: FiatCurrency;
     bitcoinUnits: PROTO.AmountUnit;
+    hasDecidedBiometrics: boolean;
```

Comment:
Why are you adding this to slice, since there is already persisted state for it? We should either use redux state for all biometrics state variables or jotai.

Reply:
It felt the same as `isOnboardingFinished` kind of prop, that is saved in the AppSettingState. So I put it here. I'll move it to where other biometrics data live.

Reply:
IMO I think that this is a good place for this state prop. I believe it fits here as well as `isOnboardingFinished` does.

Reply:
On the other hand, Bohdan is right. Now I see, that there is a `isBiometricsOptionEnabled` in jotai already with a few others. So I'll keep it with its buddies there.

Reply:
changed in 654dc5f